pkg/integrations/v4/logs/cfg_template.go

```diff
@@ -112,3 +118,16 @@ var fbConfigFormat = `{{- range .Inputs }}
 {{- if .ExternalCfg.CfgFilePath }}
 @INCLUDE {{ .ExternalCfg.CfgFilePath }}
 {{ end -}}`
+
+var fbLuaScriptFormat = `function {{ .FnName }}(tag, timestamp, record)
+    eventId = record["EventID"]
+    if {{ .ExcludedEventIds }} then
```

Comment:
As discussed, please add comments explaining what each "if block" is doing, so somebody reading the final generated function can understand what the code does:

```
- - Discard log records matching any of these conditions
if {{ .ExcludedEventIds }} then
     return -1, 0, 0
end
```

Reply:
done